### DISCUSSION TITLE
Expandible callouts work in Revealjs slides?

### DISCUSSION THREAD

**Discussion by perezp44** [quarto-dev/quarto-cli#4780] [Q&A] [answered]
I'm trying to make an expandible callout in an Revealjs slides but it don't work. There is a way that it works?
My .qmd file is:

```
---
format: revealjs
---

::: {.callout-warning  collapse="true"}
### Question
Expandible callouts does not work in Quarto Revelajs slides?
:::
```

**Comment by mcanouil** [comment, marked as answer]
As from the code base, collapse is not a valid option for `revealjs` format, at least I don't find any trace of it.

Although, the idea to have collapsible callouts in slides is questionable in my opinion.
This being said, I don't know if Quarto team wants the collapsible callouts to also work in `revealjs`, thus port the Bootstrap elements to Reveal.js.

Also note that this is mentionned in #727

**Reply by perezp44** [reply]
Thanks,
Probably is not a good idea, but I would like to have the possibility to use expandible callouts in Revealjs
Anyway, I understand that there are things more urgent and important for the Quarto team
Thanks again @mcanouil

**Comment by mcanouil** [comment]
FYI, this is tracked in #1328.
